src/utils/logger_conf.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import traceback
+import time
+import logging as log
+
+DEFAULT_FORMATTER = log.Formatter('[ %(levelname)s ] %(message)s')
+
+
+class ColorFormatter(log.Formatter):
+
+    grey = '\x1b[38;20m'
+    yellow = '\x1b[33;20m'
+    red = '\x1b[31;20m'
+    bold_red = '\x1b[31;1m'
+    reset = '\x1b[0m'
+    custom_format = '[ %(levelname)s ] %(message)s'
+
+    FORMATS = {
+        log.DEBUG: grey + custom_format + reset,
+        log.INFO: grey + custom_format + reset,
```

Comment:
please don't override info, it will use default color scheme for any theme, which is comfortable for eyes.

Reply:
removed overriding